CSS/styles.css

```diff
@@ -96,3 +104,179 @@ body {
   background-size: 100%;
   padding-bottom: 50%;
 }
+
+.works {
+  display: grid;
+}
+
+. s {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.Projectsimg {
+  width: 100%;
+  height: auto;
+}
+
+.Projectswrp {
+  background-color: #1a2236;
+  width: 100%;
+}
+
+.projtxt {
+  margin-bottom: 0.5em;
+  padding-left: 1em;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 300;
+  font-size: 20px;
+  line-height: 80%;
+  color: #fff;
+}
```

Comment:
Please correct the design to stick to Figma

here the problem is that the h2 has a margin-top so the element make a white space.
<img width="443" alt="image" src="https://user-images.githubusercontent.com/7243132/170834225-a6396fb4-1012-493b-a614-960b0706212b.png">

Figma:
<img width="443" alt="image" src="https://user-images.githubusercontent.com/7243132/170834262-f656dd73-ba77-49e5-be75-557a8d408ee2.png">